boards/vcc-gnd/yd_stm32h750vb/yd_stm32h750vb.yaml

```diff
@@ -0,0 +1,19 @@
+identifier: yd_stm32h750vb
+name: YD-STM32H750VB
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 1024
+flash: 128
+supported:
+  - gpio
+  - uart
+  - qspi
+  - sdhc
+  - rtc
+  - usb
+  - usb_device
```

Comment:
Have you run USB tests?

Reply:
Does the USB driver seem to have some issues? I tested UDC and it failed, so I decided to delete it directly

```
*** Booting Zephyr OS build v3.6.0-1948-g9965ed17614d ***
Running TESTSUITE udc_driver_test
===================================================================
I: UDC device HS: 0
START - test_udc_ep_buf
E: Unable to allocate FIFO for 0x81

    Assertion failed at WEST_TOPDIR/zephyr/tests/drivers/udc/src/main.c:332: test_udc_ep_api: (err is non-zero)
Failed to enable endpoint
 FAIL - test_udc_ep_buf in 0.083 seconds
===================================================================
START - test_udc_ep_int

    Assertion failed at WEST_TOPDIR/zephyr/tests/drivers/udc/src/main.c:377: test_udc_ep_mps: (err is non-zero)
Failed to initialize UDC driver
 FAIL - test_udc_ep_int in 0.013 seconds
===================================================================
START - test_udc_ep_iso

    Assertion failed at WEST_TOPDIR/zephyr/tests/drivers/udc/src/main.c:377: test_udc_ep_mps: (err is non-zero)
Failed to initialize UDC driver
 FAIL - test_udc_ep_iso in 0.013 seconds
===================================================================
TESTSUITE udc_driver_test failed.

------ TESTSUITE SUMMARY START ------

SUITE FAIL -   0.00% [udc_driver_test]: pass = 0, fail = 3, skip = 0, total = 3 duration = 0.109 seconds
 - FAIL - [udc_driver_test.test_udc_ep_buf] duration = 0.083 seconds
 - FAIL - [udc_driver_test.test_udc_ep_int] duration = 0.013 seconds
 - FAIL - [udc_driver_test.test_udc_ep_iso] duration = 0.013 seconds

------ TESTSUITE SUMMARY END ------

===================================================================
PROJECT EXECUTION FAILED
```

Reply:
By the way, the STM32CubeProgrammer is working properly.

```
      -------------------------------------------------------------------
                        STM32CubeProgrammer v2.16.0                  
      -------------------------------------------------------------------

ST-LINK SN  : 0F1A090029135147324D4E00
ST-LINK FW  : V2J43S7
Board       : --
Voltage     : 3.02V
SWD freq    : 4000 KHz
Connect mode: Under Reset
Reset mode  : Hardware reset
Device ID   : 0x450
Revision ID : Rev V
Device name : STM32H7xx
Flash size  : 128 KBytes
Start Address : 8000000
Device type : MCU
Device CPU  : Cortex-M7
BL Version  : 0x91



Memory Programming ...
Opening and parsing file: zephyr.hex
  File          : zephyr.hex
  Size          : 57.55 KB 
  Address       : 0x08000000 


Erasing memory corresponding to segment 0:
Download in Progress:
[==================================================] 100% 

File download complete
Time elapsed during download operation: 00:00:13.438

RUNNING Program ... 
  Address:      : 0x8000000
Application is running, Please Hold on...
Start operation achieved successfully
```